src/services/version/version.service.ts

```diff
@@ -18,9 +19,12 @@ module.exports = function (app: ImpressoApplication) {
         const sequelizeConfig = app.get('sequelize')
         const solr = app.service('simpleSolrClient')
         const isPublicApi = app.get('isPublicApi')
-
+        const cache = app.get('cacheManager')
         const [firstDate, lastDate] = await getFirstAndLastDocumentDates(solr)
         log('branch:', process.env.GIT_BRANCH, 'revision:', process.env.GIT_REVISION, 'version:', process.env.GIT_TAG)
+        const ms = await cache.get<string>(WellKnownKeys.MediaSources)
```

Comment:
@danieleguido  it's actually easier than this. To avoid repeating the same code over and over again it's easier to use the service which already return a map of all media sources:

```
const mediaSources = app.service('media-sources')
const lookup = await mediaSources.getLookup()
```